CFC_WebApp/clients/socialmedia/settings.json.sample

```diff
@@ -0,0 +1,4 @@
+{
+    "fb_app_secret":"ask sid",
```

Comment:
Yes.

I later integrated all of this into the socialmedia client’s settings. So, this file is an artifact of that.

But for future purposes, what is the best place to put these?

Sid Feygin
sid.feygin@gmail.com



> On Jun 2, 2015, at 5:04 PM, shankari <notifications@github.com> wrote:
> 
> In CFC_WebApp/clients/socialmedia/settings.json.sample <https://github.com/e-mission/e-mission-server/pull/133#discussion_r31583752>:
> 
> > @@ -0,0 +1,4 @@
> > +{
> > +    "fb_app_secret":"ask sid",
> json.sample doesn't seem to be similar to json. I presume you want to delete .json, but then if somebody takes .json.sample and copies it over, I don't think that it will work.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/e-mission/e-mission-server/pull/133/files#r31583752>.
>